test_site/pages/home.rb

```diff
@@ -8,59 +8,44 @@ class Home < SitePrism::Page
   element :welcome_header, :xpath, '//h1'
   element :welcome_message, 'body > span'
   element :go_button, '[value="Go!"]'
-  element :link_to_search_page, :xpath, '//p[2]/a'
-  # Below element takes just over 2 seconds to appear
-  element :some_slow_element, :xpath, '//a[@class="slow"]'
-  element :invisible_element, 'input.invisible'
-  element :shy_element, 'input#will_become_visible'
-  element :retiring_element, 'input#will_become_invisible'
+  element :search_page_link, :xpath, '//p[2]/a'
+  element :slow_element, :xpath, '//a[@class="slow"]'
```

Comment:
The comment might actually be useful to keep.

Reply:
This comment was incorrect, so removed.